configs/config.template.yml

```
@@ -0,0 +1,129 @@
+# Database configuration
+# Supports both individual settings and DATABASE_URL
+# DATABASE_URL takes precedence if provided
+database:
+  host: ${DATABASE_HOST:-localhost}
+  port: ${DATABASE_PORT:-5432}
+  user: ${DATABASE_USER:-postgres}
+  password: ${DATABASE_PASSWORD:-postgres}
+  dbname: ${DATABASE_DBNAME:-celebrum_ai}
+  sslmode: ${DATABASE_SSLMODE:-disable}
+  database_url: ${DATABASE_URL:-}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)